spec/javascripts/govuk-component/option-select-spec.js

```diff
@@ -0,0 +1,281 @@
+describe('GOVUK.OptionSelect', function() {
+
+  var $optionSelectHTML, optionSelect;
+
+  beforeEach(function(){
+
+    optionSelectFixture = '<div class="govuk-option-select" tabindex="0">'+
```

Comment:
Fair enough. A framework like Karma can make this easy to do eg will load all HTML fixtures and make them available to the tests - https://github.com/karma-runner/karma-html2js-preprocessor
Although you may not want to add another component to your stack..